android/app/src/main/java/com/reactnativenavigation/activities/BottomTabActivity.java

```diff
@@ -64,10 +64,10 @@ private void setupPages(ArrayList<Screen> screens) {
 
     private void setupToolbar(ArrayList<Screen> screens) {
         Screen initialScreen = screens.get(0);
-        setNavigationStyle(initialScreen);
         mToolbar.setScreens(screens);
         mToolbar.setTitle(initialScreen.title);
         setSupportActionBar(mToolbar);
+        setNavigationStyle(initialScreen);
```

Comment:
Out of curiosity, why did you move the call to `setNavigationStyle` to the bottom?